.github/workflows/deploy-plasmidseeker.yml

```diff
@@ -0,0 +1,43 @@
+# This caller workflow tests, builds, and pushes the image to Docker Hub.
```

Comment:
Fun Fact : @kapsakcj created a script, so we don't need these deploy files anymore

Reply:
Should I deleted this file from the pull request?

Reply:
That would make my life easier

Reply:
all credit goes to @SarahNadeau for the deploy YML! I just tweaked the deploy workflow a bit